app/controllers/admin/attendences_controller.rb

```diff
@@ -0,0 +1,33 @@
+class Admin::AttendencesController < Admin::BaseController
+  before_action :load_event
+
+  def index
+    @attendences=Attendence.all
+  end
+
+  def new
+    @attendence=Attendence.new
+  end
+
+  def create
+    @attendence=Attendence.new
+    @attendence.user_id=current_user.id
```

Comment:
we don't need admin to attend, so new and create actions are not necessary. besides, you can not get current_user when you're admin, if you wanted to get it must have been current_admin. please remove new and create actions for admin.